### PR TITLE
feat: Add mentor cards

### PR DESCRIPTION
**Pull request by misunchoi**
https://xd.adobe.com/view/5d144dfd-70e7-4cac-6b17-088a5b93ebcd-2b25/screen/117d6778-6199-4d99-899c-3968bc067b4d/ISMP-Website-Mentorship-FindAMentor

### CODE REVIEW

File: frontend/src/components/MentorCards/MentorCards.jsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    location: 'Los Angeles, CA',
+    education: 'U.C. Santa Barbara',
+    career: 'Project Manager',
+    skills: 'English Training, Web Development'
+  },
+  {
+    image: 'https://semantic-ui.com/images/avatar2/large/matthew.png',
+    name: 'Matthew',
+    location: 'Los Angeles, CA',
+    education: 'U.C. Berkley',
```
Comment (douglaschan32167) [inline frontend/src/components/MentorCards/MentorCards.jsx]: nit: Berkeley

File: frontend/src/components/MentorCards/MentorCards.jsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+              <Content>
+                <SubHeader>Education</SubHeader>
+                <Description>{info.education}</Description>
+                <SubHeader>Career</SubHeader>
+                <Description>{info.career}</Description>
+                <SubHeader>Skills</SubHeader>
+                <Description>{info.skills}</Description>
+              </Content>
+
+              <Button>Get in touch</Button>
```
Comment (douglaschan32167) [inline frontend/src/components/MentorCards/MentorCards.jsx]: Sorry, the image was a bit outdated. From our meetings it seems like we don't want people to be able to directly contact a mentor. Could you remove this or comment it out for now?

**Review by douglaschan32167 — CHANGES_REQUESTED** [review]
Thanks Misun! It turns out that we don't want people to directly contact mentors, so I think we can remove the get in touch button. Otherwise, it looks good!

**Comment by misunchoi** [comment]
> Thanks Misun! It turns out that we don't want people to directly contact mentors, so I think we can remove the get in touch button. Otherwise, it looks good!

**Comment by misunchoi** [comment]
sorry didn't meant to close the PR haha but made the fix!

**Review by douglaschan32167 — APPROVED** [review]
LGTM

**Pull request merged**